.github/workflows/create-pr-compose-release.yml

```
@@ -0,0 +1,43 @@
+name: Create a automatic PR on Compose Release
+description: create a Pull Request with doc changes when a new version of Docker Compose is released
+
+on:
+  repository_dispatch:
+    types: [compose_release]
+
+jobs:
+  create_compose_doc_pr:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout docs source code
+        uses: actions/checkout@v3
+
+      - name: checkout Compose source code
+        with:
+          repository: docker/compose
+          ref: ${{ github.event.inputs.release }}
+          path: ./ComposeReleaseSource
+      - name: Prepare PR
+        run: |
+          rm -rf ./_data/compose-cli/*
+          cp -rf /.ComposeReleaseSource/docs/reference/*.yaml ./_data/compose-cli/
+          rm -rf /.ComposeReleaseSource
+          sed '/compose_version:/s/\".*\"/\"${{ github.event.inputs.release }}\"/g' _config.yml
+          git add -A .
+      - name: Create PR
+        uses: peter-evans/create-pull-request@v4
```

For security purpose, pin external actions to a commit sha.

```suggestion
        uses: peter-evans/create-pull-request@f094b77505fb89581e68a1163fbd2fffece39da1 # v4.0.3
```